tests/helpers/main.ts

```diff
@@ -47,6 +47,8 @@ export const zipFixture = async function (
   const bundlerString = getBundlerNameFromConfig(config) || 'default'
   const { path: tmpDir } = await getTmpDir({
     prefix: `zip-it-test-bundler-${bundlerString}`,
+    // Cleanup the folder on process exit even if there are still files in them
+    unsafeCleanup: true,
```

Comment:
We never clean the temp folders, because there are still files left in them. With this change we do force remove the tmp folders on process exit.